src/edfi-paging-test/edfi_paging_test/api/request_client.py

```diff
@@ -207,11 +195,11 @@ def get_page(self, page: int = 1, resource: str = PERF_RESOURCE_LIST[0], page_si
             resource_name=resource,
             current_page=page,
             page_size=page_size,
-            api_response=self._get_data(next_url)
+            api_response=self._get_data(next_url),
         )
 
     def get_all(
-        self, resource: str = PERF_RESOURCE_LIST[0], page_size: int = page_size
+        self, resource: str = PERF_RESOURCE_LIST[0], page_size: Optional[int] = None
```

Comment:
Since I changed to pass the parameters through the constructor,  I had to also change how the default page size is set.